### PR TITLE
Use `credentials` instead of `keyfile` in GCS sevice

### PR DESCRIPTION
**Pull request by y-yagi**
The `keyfile` was renamed to `credentials` in `google-cloud-storage` 1.8.0.
https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-storage/CHANGELOG.md#180--2017-11-14

Although `keyfile` can still be used, but it looks like deprecate.
https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/ddf7b2a856d676316525eb581c1a4cc83ca6097b/google-cloud-storage/lib/google/cloud/storage.rb#L589...L590

Therefore, I think that should use `credentials` in newly generated applications.

Ref: https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1802

### CODE REVIEW

**Review by rafaelfranca — APPROVED** [review]

**Comment by y-yagi** [comment]
cc @georgeclaghorn

File: railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt [hunk 1]
```diff
@@ -18,7 +18,7 @@ local:
 # google:
 #   service: GCS
 #   project: your_project
-#   keyfile: <%%= Rails.root.join("path/to/gcs.keyfile") %>
+#   credentials: <%%= Rails.root.join("path/to/gcs.keyfile") %>
```
Comment (kaspth) [inline railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt]: Is the keyfile something users should best keep out of their repo? Then we could recommend `Rails.application.encrypted("config/gcs.credentials")` so they could keep it encrypted in repo.

Comment (y-yagi) [reply railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt]: The keyfile is JSON format (Google's recommended format). However, `encrypted` is supposed to use the yaml format. Therefore, I think that it is difficult to use `encrypted` as it is.

Comment (kaspth) [reply railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt]: Oh wait, this isn't the same thing. `credentials:` takes a file path so it shouldn't be encrypted, I think.

Comment (georgeclaghorn) [reply railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt]: It also takes a Hash.

Comment (georgeclaghorn) [reply railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt]: I don’t think `Rails.application.encrypted` requires that the encrypted file have any particular format, so I do think we should prefer it here.

Comment (georgeclaghorn) [reply railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt]: Scratch that. We’d still need to decode the JSON contents of the file.

**Comment by georgeclaghorn** [comment]
Sorry, I just realize you could have merged this yourself. 🤦‍♂️

**Comment by y-yagi** [comment]
Yeah, that's fine. Thank you for confirmation :)

**Pull request merged**